.github/workflows/ci.yml

```diff
@@ -1,8 +1,24 @@
 name: CI
 
-on: [push, pull_request]
+on: pull_request
```

Comment:
i think we need to keep the `push` here but only run that on the main branch, right?